beacon-chain/blockchain/execution_engine.go

```diff
@@ -233,7 +233,7 @@ func (s *Service) notifyNewPayload(ctx context.Context, postStateVersion int,
 // reportInvalidBlock deals with the event that an invalid block was detected by the execution layer
 func (s *Service) reportInvalidBlock(ctx context.Context, root, parentRoot, lvh [32]byte) error {
```

Comment:
I know this is an old name, but I think we should change it to `pruneInvalidBlock`